### PR TITLE
adding pitot

### PR DESCRIPTION
**Pull request by achevrot**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by achevrot** [comment]
@conda-forge/help-python

Thank you <3

**Comment by xoolive** [comment]
I am willing to be listed as maintainer

File: recipes/pitot/license.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (synapticarbors) [inline recipes/pitot/license.txt:1]: You can remove this license file since it appears to be included in the source distribution.

**Comment by junzis** [comment]
I am willing to be listed as maintainer.

**Comment by achevrot** [comment]
@conda-forge/help-python

There are failures on windows due to some PyTorch dependencies that are not related to our project. 
Do you have any insight into this issue?

Thank you!

**Comment by synapticarbors** [comment]
Since this is a `noarch: python` recipe, the failures on window are generally ignored and we leave it up to the user to obtain the dependency if they are on that platform.

**Pull request merged**